dmutils/content_loader.py

```diff
@@ -388,9 +498,12 @@ def get_builder(self, framework_slug, manifest):
         try:
             if framework_slug not in self._content:
                 raise KeyError
-            return ContentBuilder(self._content[framework_slug][manifest])
+            manifest = self._content[framework_slug][manifest]
         except KeyError:
             raise ContentNotFoundError("Content not found for {} and {}".format(framework_slug, manifest))
+            return ContentManifest()
```

Comment:
I've noticed that line in diffs too, just didn't realise I was the one who added it :sob: Fixed.